spec/double-tag-spec.coffee

```diff
@@ -212,3 +212,17 @@ describe "DoubleTag", ->
           editor.insertText('-span')
 
           expect(editor.getText()).toBe '<dashed-span>test</dashed-span>'
+
+    describe "undo-test", ->
+      beforeEach ->
+        editor.setText('<div>test</div>')
+
+      describe "when undo", ->
+        it "should undo in block", ->
+          editor.setCursorBufferPosition([0, 4])
+          editor.insertText('-span')
+          editor.undo()
+          # not sure why need to undo here, but in editor, one "ctrl+z" is good enough
```

Comment:
1000ms delay is only for the transact, I have tested when type thing quickly then press 'ctrl+z', it all works fine...
from the output, the first undo seems undo the change of the closing tag, the 2nd one undo the change in begnning tag.